wbooks/src/app/components/BookItem/index.tsx

```diff
@@ -1,18 +1,29 @@
 import React from 'react';
 import { View, Image, Text } from 'react-native';
 import { Book } from '@interfaces//books';
+import { TouchableOpacity } from 'react-native-gesture-handler';
+import { useNavigation } from '@react-navigation/native';
 
 import styles from './styles';
 
-function BookItem({ author, imageUrl, title }: Book) {
+interface Props {
+  book: Book;
+}
+
+function BookItem({ book }: Props) {
+  const navigation = useNavigation();
+  const { author, imageUrl, title } = book;
+
   return (
-    <View style={styles.container}>
-      <Image source={{ uri: imageUrl }} style={styles.image} />
-      <View style={styles.dataBook}>
-        <Text style={styles.textTitle}>{title}</Text>
-        <Text style={styles.textAuthor}>{author}</Text>
+    <TouchableOpacity onPress={() => navigation.navigate('BookDetail', { book })}>
+      <View style={styles.container}>
+        <Image source={{ uri: imageUrl }} style={styles.image} />
```

Comment:
Same here, add a placeholder image. You can use the `img_book_6`  for training assets.

Also, check component styles

![image](https://user-images.githubusercontent.com/8583214/108423569-c4b56000-7216-11eb-97e6-c00181cd17dd.png)

Don't use `minHeight`, missing height and width for the `image`